onnxruntime/core/mlas/lib/intrinsics/basic/min_max_elements.cpp

```diff
@@ -0,0 +1,102 @@
+/*++
+
+Copyright (c) Microsoft Corporation. All rights reserved.
+
+Licensed under the MIT License.
+
+Module Name:
+
+    min_max_elements.cpp
+
+Abstract:
+
+    This module implements the logic to find min and max elements with basic extension instructions.
+
+--*/
+
+#include "../mlasi.h"
+
+void
+MLASCALL
+MlasMinMaxF32Kernel(
+    const float* Input,
+    float* min,
+    float* max,
+    size_t N)
+{
+    if (N <= 0) {
+        *min = 0.0f;
+        *max = 0.0f;
+        return;
+    }
+
+    *min = *max = Input[0];
```

Comment:
Using the style from MlasReduceMaximumF32Kernel I would have just initialized a local to the min/max values and not needed to read Input unneccessarily here. The N <=0 case above wouldn't be necessary.